ui/widget.js

```diff
@@ -514,6 +514,16 @@ $.Widget.prototype = {
 			}
 		}
 
+		this._on( options.element, {
+			"remove": function( event ) {
+				$.each( that.classesElementLookup, function( key, value ) {
+					if ( $.inArray( event.target ) ) {
```

Comment:
@dmethvin sure is but that is fixed already this comment thread is on an outdated commit https://github.com/jquery/jquery-ui/pull/1744/files#diff-6d5657679729dec110717fc4d667b823R534